src/lennybot/check/docker_image_available.py

```diff
@@ -58,11 +58,13 @@ def _parse_image(self):
         )
 
         match = re.match(pattern, image_name)
```

Comment:
@dbalucas the match object can be `None`. It would be nice if this case was handled.

```suggestion
        match = re.match(pattern, image_name)
        if match is None:
               raise Exception(f"Given image pattern is not a valid docker image name {image_name}")
```